src/story/prologue.cpp

```diff
@@ -3,9 +3,14 @@
 #include "../battle/battle.h"
 #include "../creature/creature.h"
 #include "../creature/main_character/main_character.h"
+#include <fstream>
 
-void Prologue()
+void Prologue(MainCharacter& main_character)
 {
+	std::string file_path = "savefile.txt";
```

Comment:
I am still not convinced that this load should be in Prologue. I mean, the logical thing to do would be to have it happen right before the "Continue game" starts playing out this chapter. Prologue itself should not concern itself with whether a savefile exists or not.